tangelo/toolboxes/operators/tests/test_operators.py

```diff
@@ -107,9 +107,14 @@ def test_add(self):
         fop += FermionOperator("0^ 1", 3.)
         self.assertEqual(fop, fop_1)
 
+        # Test addition with coefficient
+        print(fop_1)
+        print(fop_1+0.)
+        self.assertEqual(0. + fop_1, fop_1)
```

Comment:
probably test the non-zero edition.

A way to do that is probably to build `f_op1` and `f_op2 = f_op1 + <that value>` and to test that `f_op2 - f_op1` is the constant you're looking for or something